nexus/src/app/instance_platform/cpu_platform.rs

```diff
@@ -846,6 +1078,76 @@ mod test {
         cpuid_leaf!(0x80000021, 0x00000045, 0x00000000, 0x00000000, 0x00000000),
     ];
 
+    // This CPUID leaf blob is some small tweaks on top of the "ideal Milan",
+    // maintaining some details that are disabled due to needed bhyve support
+    // and including Turin-specific features as supported and relevant to
+    // guests.
+    const TURIN_V1_CPUID: [CpuidEntry; 25] = [
+        cpuid_leaf!(0x0, 0x0000000D, 0x68747541, 0x444D4163, 0x69746E65),
+        cpuid_leaf!(0x1, 0x00B00F21, 0x00000800, 0xF6D83203, 0x078BFBFF),
+        cpuid_leaf!(0x5, 0x00000000, 0x00000000, 0x00000000, 0x00000000),
```

Comment:
Is leaf 5 zero here because we don't actually indicate support for monitor / mwait?

Reply:
yeah, specifically it's zero and in this list because I want to confirm the assembled profile has this leaf zeroed in addition to leaf 1 ECX `monitor` being clear.